lib/Elastica/Query/MultiMatch.php

```diff
@@ -144,13 +144,13 @@ public function setType($type)
     /**
      * Set fuzziness.
      *
-     * @param float $fuzziness
+     * @param mixed $fuzziness
```

Comment:
I like the int|string (or is it float|string)? Can $fuzzniess be a float? I think setFuzzniess must allow all possibilites. If we have an additional autoFuzziness method, why not. As a side mark here: auto is the default, so I assume this call will be rarely used.